tests/anagram.test.js

```diff
@@ -106,7 +106,7 @@ describe('anagram()', () => {
         expect(anagram("Licúa", array)).toEqual(["Lucía"]);
     });
 
-    it('run anagram with array 7', () => {
+    it('run anagram with array xyz', () => {
```

Comment:
Change description to:
```suggestion
    it('run anagram with array otherDescription', () => {
```